src/Compilers/VisualBasic/Portable/CommandLine/VisualBasicCommandLineParser.vb

```diff
@@ -399,6 +400,15 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
                 If IsScriptRunner Then
                     Select Case name
+                        Case "i", "i+"
+                            If value IsNot Nothing Then Exit Select
```

Comment:
This behavior looks inconsistent with other similar cases. For example, ```Case "doc+"``` below.
```
                         Case "doc+"
                             If value IsNot Nothing Then
                                 AddDiagnostic(diagnostics, ERRID.ERR_SwitchNeedsBool, "doc")
                             End If
```
A diagnostics is reported and the code continues. Was this error scenario tested? Is there a reason to be different?